main.js

```diff
@@ -0,0 +1,298 @@
+}
+
+// game proper (with switching players)
+
+let twoPlayerGame = function () {
+
+  // add click event to each cell on the grid
+
+  for (let square of cells) {
+    square.addEventListener('click', gaming)
+
+    function gaming() {
+
+      // prevents clicking on a filled square by throwing alert 
+      if (square.innerHTML !== "") {
+        window.alert("Please choose an empty square!")
+      } else {
+        square.innerHTML = currentPlayer.symbol;
+      }
+
+      // checks for tie       
+      tieGame()
+
+      // alternating X / O turns, checking for winning combinations
+
+      if (square.innerHTML === "X") {
+        gameWin()
+
+        if (checkedMoves.length === 3) {
+          playerStatus.innerHTML = currentPlayer.name + " wins!"
+          delayEnding()
+        } else if (fullGrid.length === 9) {
+          playerStatus.innerHTML = "It's a draw!"
+          delayEnding()
+        }
+        else {
+          currentPlayer = playerO
+          playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+        }
+
+      } else {
+        gameWin()
+
+        if (checkedMoves.length === 3) {
+          playerStatus.innerHTML = currentPlayer.name + ' wins!'
+          delayEnding()
+        } else if (fullGrid.length === 9) {
+          playerStatus.innerHTML = "It's a draw!"
+          delayEnding()
+        }
+        else {
+          currentPlayer = playerX
+          playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+        }
+
+      }
+    }
+  }
+}
+
+// ------------------------------------------------------------------------------------------------------------
+// one-player game between human and computer
+
+// starts game, disabling both player option buttons 
+
+function startGame1() {
+  playerX.name = "Player X"
+  playerO.name = "Player O"
+  currentPlayer = playerX
+  playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+  event.target.disabled = true
+  startButton2.disabled = true
+  onePlayerGame()
+}
+
+// random number generator to choose among the available spaces in the board array (by index)
+
+function randomInt(max, min) {
+  max = board.length - 1
+  min = 0
+  arrayIndex = Math.floor(min + (Math.random() * (max - min + 1)))
+  return arrayIndex
+}
+
+// main game function 
+
+board = [c0, c1, c2, c3, c4, c5, c6, c7, c8]
+
+function onePlayerGame() {
+
+  for (let square of cells) {
+    square.addEventListener('click', playGame)
+
+    // functions that use the "square" variable which (due to scope) cannot be defined outside of the onePlayerGame
+    
+    // function to eliminate unplayable spaces from the board 
+    
+    function eliminateCells() {
+      if (board.includes(square)) {
+        let indexOfItem = board.indexOf(square)
+        board.splice(indexOfItem, 1)
+        return board
```

Comment:
Here, and line 241 you don't need to return the board variable because you are mutating the original variable. You only need to return a value from a function if you need to assign the value to a new variable to be used elsewhere. Since board is a global variable, any of your functions will be able to access it.